src/main/java/org/fundacionjala/at15/pokemon/Pokemon.java

```diff
@@ -5,19 +5,26 @@
 import java.util.ArrayList;
 
 import org.fundacionjala.at15.pokemon.ID.Identifier;
+import org.fundacionjala.at15.pokemon.commands.create.IncompleteArguments;
 
 public class Pokemon extends Entity {
     private HitPoints hitPoints;
     private ArrayList<Move> movementList;
     private String pokemonName;
 
-    public Pokemon(int newMaxHP, String pokemonName) {
+    public Pokemon(int newMaxHP, String pokemonName) throws IncompleteArguments {
         this.id = Identifier.generateId(this);
         this.type = POKEMON;
+        if (newMaxHP == 0) {
```

Comment:
`<= 0` should be better

Reply:
Good observation! to avoid negative HitPoints!